common/src/main/java/com/microsoft/identity/common/internal/request/OperationParameters.java

```diff
@@ -53,6 +53,12 @@
     private String mClaimsRequestJson;
     @Expose()
     private SdkType mSdkType = SdkType.MSAL; // default value where we get a v2 id token;
+    @Expose()
+    private String mSdkVersion;
+    @Expose()
+    private String mApplicationName;
```

Comment:
This is OII - just checking this was intentional

Reply:
Synced offline , good here. Thanks Brian